tests/e2e/no_default_cr_test.go

```diff
@@ -57,5 +59,84 @@ var _ = Describe("Module Without Default CR", Ordered, func() {
 				WithArguments(kyma.GetName(), kyma.GetNamespace(), kcpClient, shared.StateReady).
 				Should(Succeed())
 		})
+
+		It("When a blocking finalizer is added to the Manifest CR", func() {
```

Comment:
To Reviewer:

This is the e2e to test no default CR module gets deleted successfully when access secret get deleted.